internal/dashboard/router/v1/obcluster_router.go

```diff
@@ -29,4 +29,5 @@ func InitOBClusterRoutes(g *gin.RouterGroup) {
 	g.POST("/obclusters/namespace/:namespace/name/:name/obzones/:obzoneName/scale", h.Wrap(h.ScaleOBServer))
 	g.DELETE("/obclusters/namespace/:namespace/name/:name/obzones/:obzoneName", h.Wrap(h.DeleteOBZone))
 	g.GET("/obclusters/:namespace/:name/essential-parameters", h.Wrap(h.ListOBClusterResources))
```

Comment:
don't forge to delete this line after front end modifies the url

Reply:
Oh yes, but the change need to be noticed by frontend app.